.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,78 @@
+name: Benchmarking
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  benchmarks:
+    name: Gate benchmarks
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
```

Comment:
Do we need this for loop if we're just testing on `ubuntu`?
Could we do
`    runs-on: ubuntu-latest`?

Reply:
BTW, these questions are also for my understanding, sorry for sounding picky!

Reply:
No, updated, thanks for the catch!